firebase-perf/src/main/java/com/google/firebase/perf/injection/components/FirebasePerformanceComponent.java

```diff
@@ -0,0 +1,31 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.perf.injection.components;
+
+import androidx.annotation.NonNull;
+import com.google.firebase.perf.FirebasePerformance;
+import com.google.firebase.perf.injection.modules.FirebasePerformanceModule;
+import dagger.Component;
+
+/**
+ * Dagger component to create FirebasePerformanceComponent Objects.
+ *
+ * @hide
```

Comment:
I know we have another bug to filter out the use-case for `@hide` annotations but do we need to add this for new APIs/Classes that we are writing?

FYI - https://github.com/firebase/firebase-android-sdk/tree/master#annotations

---

_If we want to do it later across project that is fine too. In that case, please ignore._

Reply:
We use `@hide` to hide the public methods from generating api.txt. Would you like to link the bug id? 
Yes, we will do it later across project for the annotation fix.

Reply:
Sg! b/177448055 is the bug.